src/rez/version/_version.py

```diff
@@ -41,7 +44,7 @@ def __ge__(self, other):
         return not self < other
 
     def __str__(self):
-        return "reverse(%s)" % str(self.value)
+        return "reverse(%r)" % self.value
```

Comment:
```suggestion
        return f"reverse({self.value!r})"
```
Micro-optimization, but considering how core this module is, this could actually improve the solver speed a little bit.